newIDE/app/src/LayersList/LayerRemoveDialog.js

```diff
@@ -48,13 +58,21 @@ export default class VariablesEditorDialog extends Component {
       />,
     ];
 
-    const layers = enumerateLayers(this.props.layersContainer);
+    const layers = enumerateLayers(
+      this.props.layersContainer,
+      this.props.i18n._(t`Base layer`)
+    );
     const choices = layers
       .filter(({ value }) => {
         return value !== this.props.layerRemoved;
       })
       .map(({ value, label }) => (
-        <SelectOption key={value} value={value} primaryText={label} />
+        <SelectOption
+          key={value}
+          value={value}
+          primaryText={label}
+          areUserDefinedValues
```

Comment:
should this be true only if label === 'Base layer'?
and then the SelectOption would translate the base layer text, instead of doing it yourself above, and you don't need to change the logic of the `enumerateLayers`

Reply:
It would not work for the property editor unless each choice has the boolean.